### DISCUSSION TITLE
Actividad_1 GRUPO 11

### DISCUSSION THREAD

**Discussion by pchesini** [frlp-utn-ingsoft/catedra#90] [Show and tell]
https://github.com/pchesini/vetsoft.git

**Comment by RodrigoJacznik** [comment]
Buenas!
La app funciona bien. La actividad esta aprobada

Les dejo algunos comentarios:

- Intenten estandarizar los mensajes de los commits. Por ejemplo el branch abm/medicine tiene antes del mensaje [abm/medicine] (no es una practica normal ponerle el nombre del branch antes de los commits) y el resto no.
- Presten más atención desde que branch crean los feature branch. Siempre tienen que salir del branch principal, por ejemplo abm/medicine y abm/provider no salen de main (el branch principal).

Saludos.
